### ISSUE TITLE
Web component: Content group simple - video: Modal dialog

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/carbon-for-ibm-dotcom#4411]
## Detailed description

Issue: 
1. There is no label for the modal
2. In Voiceover iPhone, the dialog role is not identified

Expected:
1. Provide appropriate label for the modal dialog using aria-label or aria-labelledby
2. Screen reader should identify the dialog role

**Comment by annawen1** [comment]
Hi @Praveen-sr, can this be tested again? I am using iphone voiceover and it reads out the video by the video title and as a button. Can you check if this issue has been resolved? Thank you!

**Comment by Praveen-sr** [comment]
@annawen1 Re tested this - Here, the Screen reader does not identify the dialog role for the modal and the label i.e. title for the modal dialog.

**Comment by Praveen-sr** [comment]
@annawen1 
Steps to reproduce 
1. Open content group simple on web components
2. On Knobs section, go to "CardCTA" 
3. Change the "CTA type" to 'Video'
4. Click on the video, it opens in modal

Here, the Screen reader does not identify the dialog role for the modal and the label i.e. title for the modal dialog.
